js/apps/admin-ui/src/components/dynamic/UserProfileAttributeListComponent.tsx

```diff
@@ -46,6 +46,10 @@ export const UserProfileAttributeListComponent = ({
 
   if (!config) return null;
 
+  const getError = (record: any, path: string) => {
+    return path.split(".").reduce((record, item) => record?.[item], record);
+  };
```

Comment:
This can probably be typed a bit stronger:

```suggestion
  const getError = (record: FieldErrors<FieldValues>, path: string) => {
    return path.split(".").reduce((record, item) => record?.[item], record);
  };
```

However, since the `errors` variable is already in the scope of this function, there is no need to pass it as an argument:

```suggestion
  const getError = (path: string) => {
    return path.split(".").reduce((errors, item) => record?.[item], errors);
  };
```

If this runs into some type errors, it is best to cast it at the last moment, so the type is preserved as much as possible.

Reply:
Done! Both arguments are defined so I have just removed them. The argument is transformed into `any` in the reduce.